### DISCUSSION TITLE
Incompatible with Terralith?

### DISCUSSION THREAD

**Discussion by Moonspire** [Vicale200/the_warden#2] [General]
Hello, after some testing I think this is incompatible with the world gen Terralith on 1.18.2.

I ran the locatebiome command in many places and it returned: "Could not find a biome of type "warden_and_sculk:sculk_biome" within reasonable distance"

I figured I should post something about it so the owner and possible users can see. If anyone has ideas on making this work with Terralith then let me know!

**Comment by Vicale200** [comment]
Well there is an option. In the config file you can find an option called sculk blobs, so you can enable this and disable sculk biome option. 

Then the deep dark/sculk biome will be generated as patches. (Ancient Cities won't generate so you need to change the tag related to it and put other biomes)

**Reply by Moonspire** [reply]
Thanks for the info, I might play around in Terralith to try to modify it to pull the structures and biomes from this into it.
